params/sheffield_hawk.R

```diff
@@ -0,0 +1,15 @@
+# !! PLEASE DO NOT EDIT THIS FILE !!
+# ** make a copy and edit that instead **
+# ________________________________________________
+# input parameters
+
+# sigma2: 0 < value < 5
+sig2 <- 4.9
+    
+# e.g. "anas_krystallinus"    
+species.name <- "sheffield_hawk"
```

Comment:
Could you please change the name of your species?

Reply:
Good spot - I've changed the name.

Reply:
Thanks Bob, that's a great name!